src/lib/web3-utils.js

```diff
@@ -0,0 +1,39 @@
+import env from '../environment'
+
+export function getUseWalletProviders() {
+  const providers = [{ id: 'injected' }, { id: 'frame' }]
+
+  if (env('FORTMATIC_API_KEY')) {
+    providers.push({
+      id: 'fortmatic',
+      useWalletConf: { apiKey: env('FORTMATIC_API_KEY') },
+    })
+  }
+
+  if (env('PORTIS_DAPP_ID')) {
+    providers.push({
+      id: 'portis',
+      useWalletConf: { dAppId: env('PORTIS_DAPP_ID') },
+    })
+  }
+
+  return providers
+}
+
+export function getUseWalletConnectors() {
+  return getUseWalletProviders().reduce((connectors, provider) => {
+    if (provider.useWalletConf) {
+      connectors[provider.id] = provider.useWalletConf
+    }
+    return connectors
+  }, {})
+}
+
+export function getNetworkName(chainId = env('CHAIN_ID')) {
+  chainId = String(chainId)
+
+  if (chainId === '1') return 'Mainnet'
```

Comment:
```suggestion
  if (chainId === '1') return 'Ethereum'
```

Or alternatively "Ethereum mainnet" :). There are lots of "mainnets" around now, so it's better to be more explicit!